jenkins/ci.suse.de/cloud-ardana8.yaml

```diff
@@ -189,7 +189,7 @@
     updates_test_enabled: false
     scenario_name: entry-scale-kvm
     tempest_filter_list: "\
-      smoke,keystone,swift,glance,cinder,neutron,nova,barbican,fwaas,vpnaas,\
+      keystone,swift,glance,cinder,neutron,nova,barbican,fwaas,vpnaas,lbaas,\
       designate,heat,ceilometer,magnum,freezer,monasca"
     qa_test_list: "iverify"
```

Comment:
has iverify actually passed reliably, ever ? if not, I would exclude it for now and look into making it reliable (e.g. we can have a Jira story for this)